src/backend/functions/users.ts

```diff
@@ -89,6 +92,10 @@ const logUserIn: ApiRouteFunction = async (_params, event) => {
     user = createdUser;
   }
 
+  const accessToken = jwt.sign(user, process.env.JWT_SECRET as string, { expiresIn: '20mins', algorithm: 'HS256' });
+
+  event.headers["accessToken"] = accessToken;
```

Comment:
I think this refers to the request event headers (correct me if I'm wrong), but we want to put it into the response headers. Also, why name the header "accessToken"? You're probably going to need to use the `buildResponse` function I wrote. Also, you'll want to research how the front-end is going to pick up the token and consider how that will impact things